README.md

```diff
@@ -1,8 +1,13 @@
 # cryptokitties-xdai-demo
 
-#### Install dependencies
+#### Clone the repository
 ```bash
-yarn
+git clone https://github.com/poanetwork/cryptokitties-xdai-demo.git
```

Comment:
Does it make sense to do `git clone --recursive` here in order to pay attention that a submodule is used?

Reply:
Sure, fixed https://github.com/poanetwork/cryptokitties-xdai-demo/pull/6/commits/e7b921fb348ec9ef974d53f2dea7dcb3176b0497